<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tricle</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QFrame" name="Outer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>41</width>
       <height>571</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="Left">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>321</width>
       <height>571</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="Mid">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QRadioButton" name="radioButtonFile">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>100</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>File</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
        <widget class="QTextEdit" name="KeyOne">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>230</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Key One</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonFolder">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>100</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Folder</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
        <widget class="QRadioButton" name="radioButtonU">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>190</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Unscramble</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
        <widget class="QPlainTextEdit" name="KeyThree">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>330</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Key Three</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="KeyTwo">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>280</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Key Two</string>
         </property>
        </widget>
        <widget class="QPushButton" name="StartButton">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>380</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="textOut">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>460</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="FileButton">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>140</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>500</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonS">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>190</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Scramble</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
        <widget class="QPushButton" name="CloseButton">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>420</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonFolders">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>100</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Folders</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
        <widget class="QFrame" name="HFrame">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>321</width>
           <height>81</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="Heading">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>0</y>
            <width>231</width>
            <height>81</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>23</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Tricl√©</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>741</width>
       <height>571</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QFrame" name="Right">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="BGshape">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>80</y>
           <width>741</width>
           <height>341</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>163</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;/&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="fList">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>741</width>
           <height>151</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>37</pointsize>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="pList">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>420</y>
           <width>741</width>
           <height>151</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>37</pointsize>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radioButtonFile</tabstop>
  <tabstop>radioButtonFolder</tabstop>
  <tabstop>radioButtonFolders</tabstop>
  <tabstop>FileButton</tabstop>
  <tabstop>radioButtonS</tabstop>
  <tabstop>radioButtonU</tabstop>
  <tabstop>KeyOne</tabstop>
  <tabstop>KeyTwo</tabstop>
  <tabstop>KeyThree</tabstop>
  <tabstop>StartButton</tabstop>
  <tabstop>CloseButton</tabstop>
  <tabstop>textOut</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>FileButton</sender>
   <signal>clicked()</signal>
   <receiver>KeyOne</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
